fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -236,7 +236,7 @@
     public static final Property COULD_NOT_STORE_PROPERTY =
             createProperty(REPOSITORY_NAMESPACE + "couldNotStoreProperty");
     public static final Property INBOUND_REFERENCES = createProperty(FCREPO_API_NAMESPACE + "PreferInboundReferences");
-    public static final Property EMBED_CONTAINS = createProperty(REPOSITORY_NAMESPACE + "EmbedResources");
+    public static final Property EMBED_CONTAINS = createProperty(OA_NAMESPACE + "PreferContainedDescriptions");
```

Comment:
This seems to already be defined on Line 242 below.

Reply:
@whikloj Good catch. I'll remove it. Thanks.

Reply:
@lsitu I'll just note that `EMBED_CONTAINS` appears in [5 files](https://github.com/fcrepo4/fcrepo4/search?utf8=%E2%9C%93&q=EMBED_CONTAINS&type=Code) and `EMBED_CONTAINED` only appears in [4 files](https://github.com/fcrepo4/fcrepo4/search?utf8=%E2%9C%93&q=EMBED_CONTAINED&type=Code). So maybe removing the `EMBED_CONTAINED` is better? At least determining what the extra use is would be good.

Reply:
Appears to be a use in an integration test. https://github.com/fcrepo4/fcrepo4/blob/f5c979dc8f1d8301aaf9081d4930e5343c86fc9b/fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersionsIT.java

Reply:
@lsitu Once the review and changes have been made, is there an interaction you can describe so that I can test locally?

Reply:
@whikloj I've changed it to use EMBED_CONTAINED as suggested by @awoods.

Reply:
@randalldfloyd I am not sure what's the best way for it. Maybe we can follow the documentation on https://wiki.duraspace.org/display/FEDORA4x/RESTful+HTTP+API for the prefer header?